package.json

```diff
@@ -7,10 +7,10 @@
   "types": "src/types.d.ts",
   "scripts": {
     "dev": "npx webpack-dev-server",
-    "build": "klap build --no-sourcemap"
+    "build": "klap build --no-sourcemap -b --browserslist"
```

Comment:
you don't need the `-b --browserlist` flag here, since you have `browserslist` in your `package.json` already.

Reply:
Awesome !
Effectively, it could be useful to explain like you've do with me in you documentation because it was not clearly explained when I've check it.